av1-rtp-spec.md

```diff
@@ -570,16 +570,19 @@ Answer SDP:
 
    The Full Intra Request (FIR) [RFC5104] RTCP feedback message allows a
    receiver to request a Decoder Refresh Point of an encoded stream.
-
-   Upon reception of FIR, the AV1 sender MUST send a new coded video sequence
-   (see section 7.5 of the [AV1] bitstream specification) as soon as possible.
+   [RFC8082] updates the definition of the Decoder Refresh Point.
+
+   Upon reception of an FIR, the AV1 sender MUST as soon as possible send a
+   new coded video sequence for every SSRC indicted in the FIR message
+   (see section 7.5 of the [AV1] bitstream specification). If an AV1 bitstream
+   contains several spatial layers without inter-layer dependencies (an "S" mode),
+   an AV1 sender MUST send a new coded video sequence for each independent
+   spatial layer.

```

Comment:
"a new coded video sequence for each independent spatial layer" is left unchanged, but I guess it is one of the source of the confusion.
Definition of a "new coded video sequence" is a bit fuzzy in S modes (are there multiple "new coded video sequences" in such case?):
"A new coded video sequence is defined to _start at each temporal unit_ which satisfies ..."
together with definition of temporal unit that imply that even if there are 3 independent spatial layers, there still only single "first" temporal unit and thus only single "new coded video sequence"
on the other side, coded video sequence must start with a KEY_FRAME that resets all codec buffers. So it would be non-trivial to produce new coded video sequence only for some independent spatial layers.

May be it will be less confusing to just remove this sentence about S-modes. Added reference to RFC8082 seems better clarifies what a decoder refresh point is.

Reply:
Yes, I think the sentence can be removed.